C/capnproto/build_tarballs.jl

```diff
@@ -0,0 +1,52 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder
+
+name = "capnproto"
+version = v"0.7.0"
+
+# Collection of sources required to build capnproto
+sources = [
+    "https://capnproto.org/capnproto-c++-$(version).tar.gz" =>
+    "c9a4c0bd88123064d483ab46ecee777f14d933359e23bff6fb4f4dbd28b4cd41",
+    "./bundled"
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/capnproto-*/
+atomic_patch -p2 ${WORKSPACE}/srcdir/patches/aligned-alloc.patch
+(
+    # Do native build to get a capnp that we can run
+    mkdir build_native && cd build_native
+    export CC=${MACHTYPE}-cc
+    export CXX=${MACHTYPE}-c++
+    export LD=${MACHTYPE}-ld
+    ../configure --host=${MACHTYPE} --with-pic \
+        lt_cv_prog_compiler_pic_works=yes \
+        lt_cv_prog_compiler_pic_works_CXX=yes
```

Comment:
I had to set these two variables because when targeting macOS these were set to "no", with not much explanation in the `config.log` :confused: